src/test/java/org/gridsuite/modification/server/utils/MatcherLineCreationInfos.java

```diff
@@ -60,6 +65,12 @@ protected MatcherLineCreationInfos(LineCreationInfos ref) {
         super(ref);
     }
 
+    public boolean matchesCurrentLimitsInfos(CurrentLimitsInfos cl1, CurrentLimitsInfos cl2) {
+        return cl1 == null && cl2 == null
+            || cl1 != null && cl2 != null && cl1.getPermanentCurrentLimit().equals(cl2.getPermanentCurrentLimit())
```

Comment:
Pb if cl1.getPermanentCurrentLimit() is null
Add parenthesis ?

Reply:
ok